test/functional/test_framework/test_node.py

```diff
@@ -245,7 +255,7 @@ def assert_start_raises_init_error(self, extra_args=None, expected_msg=None, mat
              tempfile.NamedTemporaryFile(dir=self.stdout_dir, delete=False) as log_stdout:
             try:
                 self.start(extra_args, stdout=log_stdout, stderr=log_stderr, *args, **kwargs)
-                self.wait_for_rpc_connection()
+                timeout = self.wait_for_rpc_connection()
```

Comment:
This `timeout` variable used? If not, `wait_for_rpc_connection`  does not need to return this.

Reply:
right! its not used. will fix it.